project.yml

```diff
@@ -130,8 +130,8 @@ targets:
       identifier: "org.datepollsystems.waiterrobot.beta"
       displayName: "lava.kellner.team"
       deeplinkDomain: "lava.kellner.team"
-      apiBase: "https://lava.kellner.team/api"
       versionSuffix: "lava"
+      allowedHosts: "my.kellner.team,lava.kellner.team"
```

Comment:
Fine but on android I use "*" so that the lava version can also be tested with a local backend :)